storage/store_test.go

```diff
@@ -1348,13 +1357,18 @@ func (fq *fakeRangeQueue) MaybeRemove(rng *Range) {
 // TestMaybeRemove tests that MaybeRemove is called when a range is removed.
 func TestMaybeRemove(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	store, _, stopper := createTestStore(t)
+	store, _, stopper := createTestStoreWithoutStart(t)
 	defer stopper.Stop()
```

Comment:
Hmm, I wasn't totally sure, but I guess it's safer to do `defer` here since `stopper` is passed to ` store.Bootstrap`.